packages/zowe-explorer-api/src/profiles/ProfilesCache.ts

```diff
@@ -316,10 +316,15 @@ export class ProfilesCache {
     }
 
     // This will retrieve the base profile from imperative
-    public async fetchBaseProfile(): Promise<imperative.IProfileLoaded | undefined> {
+    public async fetchBaseProfile(profileName?: string): Promise<imperative.IProfileLoaded | undefined> {
         const mProfileInfo = await this.getProfileInfo();
         const baseProfileAttrs = mProfileInfo.getDefaultProfile("base");
-        if (baseProfileAttrs == null) {
+        const isUsingTokenAuth = (profName: string): boolean =>
+            mProfileInfo.getTeamConfig().api.secure.securePropsForProfile(profName).includes("tokenValue");
```

Comment:
Is this arrow function necessary? It's only used once in the `if` statement below.

Reply:
Removed in 393169eeafd84442b52f4fcd03ca058b0124b398